lib/template/page.ejs

```diff
@@ -16,5 +16,6 @@
 </body>
 <script src="<%- asset.vue %>"></script>
 <script src="<%- asset.vueStrap %>"></script>
+<script>const baseUrl = '<%= baseUrl %>'</script>
```

Comment:
Use block to differentiate this:

```html
<script>
    const baseUrl = '<%- baseUrl %>';
</script>
```

If we don't need to escape it (`<%=`), let's keep it consistent (`<%-`).

Reply:
Trailing semicolon!